engine/common/version/version_control.go

```diff
@@ -40,6 +40,7 @@ var NoHeight = uint64(0)
 var defaultCompatibilityOverrides = map[string]struct{}{
 	"0.37.17": {},
 	"0.37.18": {},
+	"0.37.22": {},
```

Comment:
assuming that the tag would be v0.37.22 (the last tag on v0.37 currently is v0.37.21)